manifests/role/ansible_master.pp

```diff
@@ -40,6 +40,7 @@
   $playbooks_source = 'https://github.com/openshift/openshift-ansible.git',
   $playbooks_version = 'master',
   $manage_ansible_cfg = true,
+  String $inventory_file = '/etc/ansible/hosts',
```

Comment:
`Stdlib::Absolutepath`, but in adding types you make the module incompatible with Puppet 3 which will need the following:

* Update documentation, CI and metadata accordingly
* Preferrably replace uses of `validate_*` with Puppet types

Reply:
Keep Puppet 3 compatibility for now.